### PR TITLE
[TTAHUB-1047] Raise test coverage threshold to 75% on backend

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change
Raise test coverage threshold to more reasonable levels (pre-breakout of tests into their own pockets)

## How to test
CI passes

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1047

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -171,14 +175,20 @@
       "<rootDir>/src/*/coverage/",
       "<rootDir>/src/coverage/",
       "<rootDir>/src/index.js",
-      "<rootDir>/src/worker.js"
+      "<rootDir>/src/worker.js",
```
Comment (nvms) [inline package.json:178]: these two have the `.ts` extension now

Comment (nvms) [reply package.json:178]: (I probably should have caught this in the typescript PR, whoops)

**Review by nvms — APPROVED** [review]
this is great, thank you for taking the time to write all these tests

**Comment by GarrettEHill** [comment]
Most of this looks great. But the use of mocks to prevent interaction with the real database breaks our ability to use raw queries for more complex integrations with the database. We need our test to test the system, not bypass the potentially weakest points of it.

**Pull request merged**